openfl/transport/grpc/aggregator_server.py

```diff
@@ -157,27 +157,7 @@ def GetTasks(self, request, context):  # NOQA:N802
         tasks, round_number, sleep_time, time_to_quit = self.aggregator.get_tasks(
             request.header.sender
         )
-        if tasks:
-            if isinstance(tasks[0], str):
-                # backward compatibility
-                tasks_proto = [
-                    aggregator_pb2.Task(
-                        name=task,
-                    )
-                    for task in tasks
-                ]
-            else:
-                tasks_proto = [
-                    aggregator_pb2.Task(
-                        name=task.name,
-                        function_name=task.function_name,
-                        task_type=task.task_type,
-                        apply_local=task.apply_local,
-                    )
-                    for task in tasks
-                ]
-        else:
-            tasks_proto = []
+        tasks_proto = [aggregator_pb2.Task(name=task) for task in tasks]
```

Comment:
these 20 lines are same as the one line, and covers all edge cases (note that removal of internal if-else is partly because interactive API is defunct)